src/libraries/System.Runtime/tests/System/TimeZoneInfoTests.cs

```diff
@@ -2321,6 +2321,50 @@ public static void TestNameWithInvariantCulture()
 
         }
 
+        [Theory]
+        [PlatformSpecific(TestPlatforms.Browser)]
+        [InlineData("America/Buenos_Aires", "America/Argentina/Buenos_Aires")]
+        [InlineData("America/Catamarca", "America/Argentina/Catamarca")]
+        [InlineData("America/Cordoba", "America/Argentina/Cordoba")]
+        [InlineData("America/Jujuy", "America/Argentina/Jujuy")]
+        [InlineData("America/Mendoza", "America/Argentina/Mendoza")]
+        [InlineData("America/Indianapolis", "America/Indiana/Indianapolis")]
+        public static void TestTimeZoneIdBackwardCompatibility(string oldId, string currentId)
+        {
+            TimeZoneInfo oldtz = TimeZoneInfo.FindSystemTimeZoneById(oldId);
+            TimeZoneInfo currenttz = TimeZoneInfo.FindSystemTimeZoneById(currentId);
+
+            Assert.Equal(oldtz.StandardName, currenttz.StandardName);
+            Assert.Equal(oldtz.DisplayName, currenttz.DisplayName);
+        }
+
+        [Theory]
+        [PlatformSpecific(TestPlatforms.Browser)]
```

Comment:
afaik, the general practice is to not restrict test cases for particular platform, when they pass on other platforms.